packages/sdk/package.json

```diff
@@ -45,7 +47,7 @@
     "debug": "^4.3.4",
     "ethers": "^6.8.1",
     "graphql-request": "^6.0.0",
-    "iexec": "^8.5.0",
+    "iexec": "^8.6.0",
```

Comment:
Do we need to add a quick note on the changelog for this iexec-sdk bump?

Reply:
well... we didn't for the other upgrades of the iexec SDK, not sure what to do...

Reply:
No problem then, I wasn't sure :)